Mathlib/Data/Set/Card.lean

```diff
@@ -76,6 +76,11 @@ theorem encard_eq_coe_toFinset_card (s : Set α) [Fintype s] : encard s = s.toFi
   have h := toFinite s
   rw [h.encard_eq_coe_toFinset_card, toFinite_toFinset]
 
+@[simp] theorem toENat_cardinalMk (s : Set α) : (Cardinal.mk s).toENat = s.encard := rfl
+
+@[simp] theorem coe_fintypeCard (s : Set α) [Fintype s] : (Fintype.card s : ℕ∞) = s.encard := by
```

Comment:
Is there a parenthesis because you wanted to write `some (Fintype.card s : ℕ∞)` and then you dropped it?

Reply:
No - but I've removed the parentheses :)